keyboards/maxr1998/phoebe/phoebe.h

```diff
@@ -0,0 +1,21 @@
+#ifndef PULSE4K_H
+#define PULSE4K_H
```

Comment:
```suggestion
```

Reply:
I also used the wrong name here anyway, oops